File: DSA/Array/0026_Remove_Duplicates_from_Sorted_Array/solution.py
# 0026. Remove Duplicates from Sorted Array
# Difficulty : Easy
# Tags       : array, two-pointers
# Solved on  : 2026-04-28
# LeetCode   : https://leetcode.com/problems/remove-duplicates-from-sorted-array/

class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        nums[:] = list(dict.fromkeys(nums))
        return len(nums)

File: DSA/Array/0026_Remove_Duplicates_from_Sorted_Array/php/solution.php
// 0026. Remove Duplicates from Sorted Array
// Difficulty : Easy
// Tags       : array, two-pointers
// Solved on  : 2022-06-14
// Attempt    : #1
// LeetCode   : https://leetcode.com/problems/remove-duplicates-from-sorted-array/

class Solution {

    /**
     * @param Integer[] $nums
     * @return Integer
     */
    function removeDuplicates(&$nums) {
        
        $arr=array();
        foreach($nums as $num)
        {
            if(!in_array($num,$arr))
            {
                array_push($arr,$num);
            }
        }
        $nums=$arr;
        return count($nums);
    }
}